docs/organizer-guide/meetups/faq-meetups.rst

```diff
@@ -1,10 +1,10 @@
 Meetups FAQ
 ===========
 
-The following list of questions and answers cover some issues that you might
+The following questions and answers cover some situations that you might
 encounter when you want to `start a meetup <https://www.writethedocs.org/organizer-guide/meetups/starting/>`_.
 
-Can Write the Docs cover the Meetup.com fees?
+Does Write the Docs cover the Meetup.com fees?
 ---------------------------------------------
```

Comment:
```suggestion
----------------------------------------------
```

Reply:
These are extremely annoying.

Reply:
Ah, thanks for catching that! I haven't worked with reST much recently and forgot the heading lines need to match length.